src/_ui/layout/mainRoundAbout.svelte

```diff
@@ -25,8 +10,8 @@
   }
 </script>
 
-{#await getPictures()}
-  <div class="flex justify-center h-screen">
+{#if false}
```

Comment:
entfern dann alles komplett und nicht mit if false

Reply:
Alright, war mir nicht sicher, müsste mir erst angucken, wie man es richtig mit dem await macht. Das ist eig ne ganz gute Idee, deswegen: Erstmal drin lassen und am Ende gucken, ob wir es rausnehmen oder soll ich es ganz rausnehmen und wir machen es nachher rein, wenn wir es reinmachen wollen.

Reply:
Was meinst du im parent

Reply:
also in der svelte-Klasse die die mainroundabout called

Reply:
Alright, also hier alles nicht benutzte rausnehmen, ne?

Reply:
Einfach das was immer true ist reinballern und das if false rauslöschen